sphinx/source/docs/releases/0.12.6.rst

```diff
@@ -58,3 +58,19 @@ Old                            New
 Since these attributes may be present in user code (e.g CustomJS callbacks
 or extensions), the old names will continue to work for some time, with a
 deprecation warning in the JS console.
+
+New signaling API
+~~~~~~~~~~~~~~~~~
+
+Previously we used Backbone events for communication between bokehjs' objects.
+This was changed to a new type-safe API. This change mostly affects contributors
+to bokehjs and writers of extensions.
+
+===================================== ==============================================
+Old                                   New
+===================================== ==============================================
+
+``@listenTo(obj, "change", fn)``      ``@connectTo(obj.change, fn)``
+``@listenTo(obj, "change:some", fn)`` ``@connectTo(obj.properties.some.change, fn)``
+``obj.trigger("change", arg)``        ``obj.change.emit(arg)``
+``obj.trigger("change:some", arg)``   ``obj.properties.some.change.emit(arg)``
```

Comment:
> I would still prefer connect_to for stylistic consistency

I forgot about this. I did rename to `connetTo` prior your earlier comment.

> I think attr would be a less confusingthign name than some.

Fine with me. I use `some` as a placeholder, like `foo`.

> Have versions of the old functions been added back with deprecation warnings?

Yes, you can still use `@trigger("change:some")`, though there will be deprecation warning.